src/net/xhr/common.ts

```diff
@@ -0,0 +1,88 @@
+export const XMLHttpRequestResponseType = {
```

Comment:
change the name of this file to `xhr.ts`. common.ts will be confusing per architecture we have.

Reply:
If those values will be used everywhere, move it to `shared` folder.

If they are gonna use globally on the project as type, we should move those to `types/` folder